### PR TITLE
feat: implement blockquote

### LINKED ISSUE #68: Block Quote
See https://www.stewart.com/en/state-pages/alabama-agents.html

requires importer updates as well

### PR DESCRIPTION
**Pull request by usman-khalid**
Fix #68 

Test URLs:
- Before: https://main--stewart--hlxsites.hlx.page/library/blocks/blockquote
- After: https://feature-blockquote--stewart--hlxsites.hlx.page/library/blocks/blockquote

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -59,6 +59,29 @@ export function createElement(tagName, props, html) {
...
+  oldElement.parentNode.replaceChild(newElement, oldElement);
+
+  if (newElementClassNames) {
+    newElement.classList.add(...newElementClassNames);
+  }
+
+  return newElement;
+};
+
+export default {
```
Comment (shsteimer) [inline scripts/scripts.js:81]: why?

File: blocks/blockquote/blockquote.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  });
+};
+
+const wrapQuote = (block) => {
+  const quote = block.querySelector(`.${classNames.blockquoteContent} > div`);
+  const blockQuoteElement = replaceElementTagName(quote, 'blockquote');
+  blockQuoteElement.innerHTML = `<p>${quote.innerHTML}</p>`;
+};
+
+export default function decorate(block) {
```
Comment (shsteimer) [inline blocks/blockquote/blockquote.js:30]: I don't think I like the authoring format here, having 3 rows seems a bit too complex with no real benefit. General, for things like this, I do a 1x2 tables:

* col 1: the image
* col 2: the text content, using formatting tools in word to designate the various areas (eg. headline, bold, quote, etc.)

If you need to reformat the DOM to be 3 divs (or whatevs) it's fine to do that in the block decorate js, but lets make sure the authoring format is optimized for ease of authors, not final Dom structure or ease of development.

Comment (usman-khalid) [reply blocks/blockquote/blockquote.js]: Updated.

File: scripts/scripts.js (hunk 1 above)
Comment (usman-khalid) [reply scripts/scripts.js]: This was originally being exported from its own file — made it over here with the copy pasta. The function is not needed anymore, so it's been removed entirely.

**Review by shsteimer — APPROVED** [review]

**Pull request merged**